cmd/cli/main.go

```diff
@@ -89,9 +95,8 @@ func main() {
 					if err != nil {
 						return fmt.Errorf("cannot create hints: %w", err)
 					}
-
 					fmt.Println("Running....")
-					runner, err := runnerzero.NewRunner(program, hints, proofmode, maxsteps)
+					runner, err := runnerzero.NewRunner(program, hints, proofmode, maxsteps, layoutName)
```

Comment:
the `layoutName` being a string sounds risky. we should maybe parse it to some enum and throw an error if needed. then again `GetLayout` does throw an error so i guess thats ok.

Reply:
yeah I suppose we coul ddo that and map a value from the flag to an enum value, but I guess it's sufficient for now